doc-site/docs/architecture/distributed_sequencer.md

```diff
@@ -0,0 +1,1437 @@
+ - Each node delegates all transactions to which ever node it determines as the current coordinator based on its latest knowledge of "current block"
+ - The delegate node will accept the delegation if its awareness of current block also results in it being chosen by the selector function.  Otherwise, the delegate node rejects the delegation and includes its view of current block number in the response
+ - On receiving the delegation rejection, the sender node can determine if it is ahead or behind (in terms of block indexing) the node it had chosen as delegate.  
+ - If the sender node is ahead, it continues to retry the delegation until the delegate node finally catches up and accepts the delegation
+ - If the sender node is behind, it waits until its block indexer catches up and then selects the coordinator for the new range
+ - Coordinator node will continue to coordinate ( send endorsement requests and submit endorsed transactions to base ledger) until its block indexer has reached a block number that causes the coordinator selector to select a different node.
+ - at that time, it waits until all dispatched transactions are confirmed on chain, then delegates all current inflight transactions to the new coordinator.
+ - if the new coordinator is not up to date with block indexing, then it will reject and the delegation will be retried until it catches up. 
+ - while a node is the current selected coordinator, it sends endorsement requests to every other node for every transaction that it is coordinating
+ -  The endorsement request includes the name of the coordinator node
+ - Each endorsing node runs the selector function to determine if it believes that is the correct coordinator for the current block number
+ - if not, then it rejects the endorsement and includes its view of the current block number in the rejection message
+ - when the coordinator receives the rejection message, it can determine if it is ahead or behind the requested endorser
+ - if the coordinator is ahead, it retries the endorsement request until the endorser catches up and eventually endorses the transaction
+ - if the coordinator is behind, then it waits until its block indexer reaches the next range boundary and delegates all inflight transactions to the new coordinator
+
+
+
+### Sender's responsibility
+
+The sender node for any given transaction remains ultimately responsible for ensuring that transaction is successfully confirmed on chain or finalized as failed if it is not possible to complete the processing for any reason.  While the coordination of assembly and endorsement is delegated to another node, the sender continues to monitor the progress and is responsible for initiating retries or re-delegation to other coordinator nodes as appropriate.
```

Comment:
> or re-delegation

Again we have the inconsistency here. This paragraph states that re-delegation is the sender's responsibility, but as discussed in comments above we seem to have split this responsibility between sender and coordinator.

As I've mentioned above, I think it would be a simpler to understand spec if it was always the sender's responsibility as this paragraph indicates.